x/ibc/03-connection/exported/exported.go

```diff
@@ -2,14 +2,13 @@ package exported
 
 import (
 	commitmentexported "github.com/cosmos/cosmos-sdk/x/ibc/23-commitment/exported"
-	ibctypes "github.com/cosmos/cosmos-sdk/x/ibc/types"
 )
 
 // ConnectionI describes the required methods for a connection.
 type ConnectionI interface {
-	GetState() ibctypes.State
 	GetID() string
 	GetClientID() string
+	GetState() int32
```

Comment:
why not just declare `type State int32` in exported?

Reply:
I could but I'd prefer having a single `.proto` file per ICS.